middleware/kubernetes/README.md

```diff
@@ -180,35 +218,6 @@ TODO(miek|...): below this line file bugs or issues and cleanup:
 
 ## Implementation Notes/Ideas
```

Comment:
Move all this stuff to another file or even to a GitHub issue. Some of it doesn't apply anymore (we do already handle reverse lookups for example).